modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/select_account_entry.jsp

```diff
@@ -99,39 +99,29 @@ if (selectAccountEntryManagementToolbarDisplayContext.isSingleSelect()) {
 	</liferay-ui:search-container>
 </clay:container-fluid>
 
-<c:choose>
-	<c:when test="<%= selectAccountEntryManagementToolbarDisplayContext.isSingleSelect() %>">
-		<aui:script>
-			Liferay.Util.selectEntityHandler(
-				'#<portlet:namespace />selectAccountEntry',
-				'<%= HtmlUtil.escapeJS(liferayPortletResponse.getNamespace() + "selectAccountEntry") %>'
-			);
-		</aui:script>
-	</c:when>
-	<c:otherwise>
-		<aui:script use="liferay-search-container">
-			var searchContainer = Liferay.SearchContainer.get(
-				'<portlet:namespace />accountEntries'
-			);
+<c:if test="<%= !selectAccountEntryManagementToolbarDisplayContext.isSingleSelect() %>">
```

Comment:
The `openSelectionModal` API supports `multiple: true`, so wondering why this is also needed. 

Maybe this is related to what was reported as [SelectedData property in openSelectionModal only affects single selection with Choose button](https://issues.liferay.com/browse/LPS-122192)

Reply:
> The openSelectionModal API supports multiple: true, so wondering why this is still needed.

@jbalsas The implementation of multiple selection behavior is still completely in JSPs. `multiple: true` in modal API just ensures that it doesn't get in the way, doesn't close modal on first `.selector-button` click.

I do think that we can be opinionated about this and implement the behavior in `openSelectionModal`. We would need to keep track of selected items and pass it as CSV to `onSelect` callback:
1. Looking at [Lexicon](https://liferay.design/lexicon/core-components/modals/#multiple-selection-inside-a-modal), we may want to do it as an integration with search container. This would imply extending modal API with `searchContainerId`.
2. Alternatively, we could have a more general solution, with adding click events on any item with  `.selector-button`. This is currently tied to single selection, but there is no reason why it couldn't be for multiple as well. This may require changing current usage implementations, as we are not placing `.selector-button`s on multiple selections. 

> Maybe this is related to what was reported as SelectedData property in openSelectionModal only affects single selection with Choose button

Yeah, it is related. I see these solutions:
1. If we want a general solution, we need a different API to determine what needs to be disabled. For example, we could expect an array of selector IDs in `selectedData`, run `getElementById` for each and disable them.
1. If we have integration with search container, we could use its API (`searchContainer.select.toggleRow`)
1. If we start using `.selector-button` for multiple selection, this ticket will be solved as a side-effect. [This block of code](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js#L391-L405) will disable it all.